README.md

```diff
@@ -525,6 +525,9 @@ You can also change the country e.g. `sold_in_uk=1`, `sold_in_de=1`, etc.
 * [Public Datasets](https://github.com/caesar0301/awesome-public-datasets) *(Datasets beyond just JSON)*
 * [Style Guides](https://github.com/kciter/awesome-style-guide) *(Programming Languages, Platforms, Frameworks)*
 
+## Consumer Complaint Database
```

Comment:
I would just put this under "Government" e.g.

`* [Consumer Complaint Database](http://data.consumerfinance.gov/api/views.json)`

Below "Corporate Consumer Contact Directory"
![image](https://cloud.githubusercontent.com/assets/398230/19120776/ed284fb8-8ad8-11e6-86ba-b18bc575dbe3.png)